### DISCUSSION TITLE
Endlessly refreshing after signup

### DISCUSSION THREAD

**Discussion by Topener** [Budibase/budibase#4063] [Help] [answered]
After I sign up I get into a loop of endlessly refreshing, I've included the URL and the console of my firefox installation in case it helps anything! 

https://user-images.githubusercontent.com/1898949/149755890-2131a270-698c-43c0-ad38-42c6fbfbc9a3.mov

**Comment by shogunpurple** [comment, marked as answer]
Hi @Topener - can you try clearing your cache and cookies in your browser? Does the problem still occur?

**Reply by Topener** [reply]
Well, there we go, clearing cookies fixed it. After logging back in it seems fine. 

Of course I should've tried it, but I guess this might be something for the buglog :)
